corehq/apps/hqwebapp/templates/hqwebapp/base.html

```diff
@@ -2,7 +2,7 @@
 {% get_current_language as LANGUAGE_CODE %}
 <html lang="{{ LANGUAGE_CODE }}">
     <head>
-        {% if not skip_google_tag_manager %}
+        {% if not enterprise_mode %}
```

Comment:
I think a better flag to use would be the `is_saas_environment` flag which is only True on prod.

Reply:
@snopoke but we would want this on india server as well where we would have non enterprise projects?

Reply:
I don't think we want this on india server since we aren't marketing india server at all. If that changes we can always add it back.

Reply:
Updated.